--- src/main/java/io/github/carlos_emr/carlos/managers/AuditLogManager.java
+++ src/main/java/io/github/carlos_emr/carlos/managers/AuditLogManager.java
@@ -104,11 +104,12 @@
         SimpleDateFormat formatter2 = new SimpleDateFormat("yyyy-MM-dd HH:mm:ss");
         SimpleDateFormat formatter3 = new SimpleDateFormat("yyyyMMddHHmmss");

-        String filename = outputDirectory + "/OSCAR_AUDIR_LOG_PURGE_FILE_" + formatter3.format(endDateToPurge) + ".sql";
+        StringBuilder filename = new StringBuilder(outputDirectory);
+        filename.append("/OSCAR_AUDIR_LOG_PURGE_FILE_").append(formatter3.format(endDateToPurge)).append(".sql");

         List<String> vars = new ArrayList<>();
         vars.add(mysqldump);
         vars.add("--user");
         vars.add(user);
         vars.add("-w");

@@ -119,7 +120,7 @@
         vars.add("-t");

         vars.add("--result-file");
-        vars.add(filename);
+        vars.add(filename.toString());

         vars.add(dbName);
         vars.add("log");
@@ -131,7 +132,7 @@
         try {
             String s = null;

-            ProcessBuilder pb = new ProcessBuilder(vars.toArray(new String[vars.size()]));
+            ProcessBuilder pb = new ProcessBuilder(vars);
             if (password != null) {
                 pb.environment().put("MYSQL_PWD", password);
             }
@@ -161,7 +162,7 @@
             throw new Exception("Error running mysqldump command. Received an exit value of " + exitValue);
         }

-        logger.info("Backed up audit log which will be purged to " + filename);
+        logger.info("Backed up audit log which will be purged to " + filename.toString());

         LogAction.addLogSynchronous(loggedInInfo, "AuditLogManager.purgeAuditLog", formatter2.format(endDateToPurge));
